models/cut_semantic_mask_model.py

```diff
@@ -34,6 +34,7 @@ def modify_commandline_options(parser, is_train=True):
 
         parser.add_argument('--contrastive_noise', type=float, default=0.0, help='noise on constrastive classifier')
         parser.add_argument('--lambda_sem', type=float, default=1.0, help='weight for semantic loss')
+        parser.add_argument('--supervised_eval_f_s', action="store_true", help='Measure f_s accuracy on validation set')
```

Comment:
`--supervised_eval_f_t` ?

Reply:
I was planning to evaluate f_s as well as f_t to compare the two, this can be done in a future PR